src/android/StatusBar.java

```diff
@@ -36,12 +36,24 @@
 import org.apache.cordova.CordovaWebView;
 import org.apache.cordova.LOG;
 import org.apache.cordova.PluginResult;
+import org.jetbrains.annotations.NonNls;
```

Comment:
is this import really safe? Is it included in the Android SDK?

Reply:
On a basic gradle project, this import fails, but in a cordova-android project this import works as is.

But `org.jetbriains` is not existent in the Android documentation, and I don't see how we are importing `'org.jetbrains:annotations:23.0.0'` (via [jetbrains documentation](https://github.com/JetBrains/java-annotations#using-the-annotations)) so this import might be relying on a third-party dependency implementation detail.

I wonder if we should be explicitly declaring this dependency in gradle.

Reply:
I went ahead and removed the import and annotations usage of `NonNls`.
I added it only to remove the IDEs annoyance for localization on these items.
It is not necessary to keep.